library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt

```diff
@@ -117,7 +118,7 @@ internal class DonutProgressLine(
 
     private fun updatePathEffect() {
         val pathLen = PathMeasure(path, false).length
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/3f8afbf9c8b622c5d946604681947fabe1098940/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L120"></span>Parentheses in (pathLen.toDouble() * length) are unnecessary and can be replaced with: pathLen.toDouble() * length</td>
    </tr>
  </tbody>
</table>